packages/app/src/server/service/slack-command-handler/keep.js

```diff
@@ -137,10 +139,27 @@ module.exports = (crowi) => {
     return result;
   };
 
+  /**
+   * Convert slack ID to growi user if slack ID is found in messages
+   * @param {*} messages
+   */
+  handler.messagesWithGrowiUser = async function(messages) {
+    await Promise.all(messages.map(async(message) => {
+      const growiUser = await User.findUserBySlackId(message.user);
+      if (growiUser) {
+        message.user = `${growiUser.name} (@${growiUser.username})`;
+      }
+      else {
+        message.user = `This slack ID is not registered (${message.user})`;
+      }
+    }));
+  };
```

Comment:
This part has a query optimization issue(we can reduce the number of query by fetching all corresponding growi users at once). So, please implement `findUsersBySlackIds` method that takes an array of slack ID as a parameter and returns an array of growi users, and call it here.

The logic of `if-else` statement looks great.